<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AXION - light &amp; fast duplicate photo finder</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ax_icon_bare.png</normaloff>ax_icon_bare.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>48</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Axion_bar.PNG</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>검색 경로</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="0">
        <widget class="QPushButton" name="path_btn_8">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="path_btn">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="fileline">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="fileline_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="path_btn_2">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="path_btn_4">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="path_btn_5">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="path_btn_3">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="path_btn_6">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="path_btn_9">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="path_btn_10">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="path_btn_7">
         <property name="text">
          <string>경로 추가</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="fileline_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="fileline_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="fileline_5"/>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="fileline_6"/>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="fileline_7"/>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="fileline_8"/>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="fileline_9"/>
       </item>
       <item row="9" column="2">
        <widget class="QLineEdit" name="fileline_10"/>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QCheckBox" name="checkBox_4">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QCheckBox" name="checkBox_5">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QCheckBox" name="checkBox_6">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QCheckBox" name="checkBox_7">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QCheckBox" name="checkBox_8">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QCheckBox" name="checkBox_9">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QCheckBox" name="checkBox_10">
         <property name="text">
          <string>하위 경로 탐색</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="start_btn">
           <property name="text">
            <string>중복 검사</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <addaction name="config"/>
    <addaction name="log"/>
    <addaction name="information"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="config">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>config.png</normaloff>config.png</iconset>
   </property>
   <property name="text">
    <string>설정</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="log">
   <property name="icon">
    <iconset>
     <normaloff>log.png</normaloff>log.png</iconset>
   </property>
   <property name="text">
    <string>로그</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="information">
   <property name="icon">
    <iconset>
     <normaloff>infor.png</normaloff>infor.png</iconset>
   </property>
   <property name="text">
    <string>정보</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
